src/main/scala/groundtest/NastiTest.scala

```diff
@@ -19,7 +18,7 @@ class NastiGenerator(id: Int)(implicit val p: Parameters) extends Module
     val mem = new NastiIO
   }
 
-  val mifDataBytes = mifDataBits / 8
+  val nastiBytes = nastiXDataBits / 8
```

Comment:
It's really the same as nastiXStrobeBits. But I guess we could also have a nastiXDataBytes.